projectforge-business/src/main/kotlin/org/projectforge/business/poll/PollDO.kt

```diff
@@ -104,17 +104,9 @@ open class PollDO : DefaultBaseDO() {
         if (accessUserIds?.contains(currentUserId) == true) {
             assignmentList.add(PollAssignment.ACCESS)
         }
-        if (!this.fullAccessUserIds.isNullOrBlank()) {
-            val accessUserIds = this.fullAccessUserIds!!.split(", ").map { it.toInt() }.toIntArray()
-            if (accessUserIds.contains(currentUserId)) {
-                assignmentList.add(PollAssignment.ACCESS)
-            }
-        }
-        if (!this.attendeeIds.isNullOrBlank()) {
-            val attendeeUserIds = this.attendeeIds!!.split(", ").map { it.toInt() }.toIntArray()
-            if (attendeeUserIds.contains(currentUserId)) {
-                assignmentList.add(PollAssignment.ATTENDEE)
-            }
+        val attendeeUserIds = toIntArray(this.attendeeIds)
+        if (attendeeUserIds?.contains(currentUserId) == true) {
+            assignmentList.add(PollAssignment.ATTENDEE)
```

Comment:
AccessUser check removed ohne es gefixt wieder einzufügen

Reply:
`        
val currentUserId = ThreadLocalUserContext.userId!!
        val assignmentList = mutableListOf<PollAssignment>()
        if (currentUserId == this.owner?.id) {
            assignmentList.add(PollAssignment.OWNER)
        }
        val accessUserIds = toIntArray(this.fullAccessUserIds)
        if (accessUserIds?.contains(currentUserId) == true) {
            assignmentList.add(PollAssignment.ACCESS)
        }
        val attendeeUserIds = toIntArray(this.attendeeIds)
        if (attendeeUserIds?.contains(currentUserId) == true) {
            assignmentList.add(PollAssignment.ATTENDEE)
        }
        if (assignmentList.isEmpty())
            assignmentList.add(PollAssignment.OTHER)

        return assignmentList
        `